src/shady-css/common.ts

```diff
@@ -47,17 +47,23 @@ export interface Stylesheet {
   /**
    * The list of rules that appear at the top level of the stylesheet.
    */
-  rules: Rule[]
+  rules: Rule[];
+
+  range: Range;
```

Comment:
range is simpler, and this way there's no confusion between analyzer's SourceRanges and these simple Ranges